bubblewrap.c

```diff
@@ -2814,6 +2847,10 @@ main (int    argc,
   /* We block sigchild here so that we can use signalfd in the monitor. */
   block_sigchild ();
 
+  /* We block other signals here to avoid leaving an orphan. */
+  if (opt_signal_propogate)
+    gate_signals (SIG_BLOCK, &sigmask);
```

Comment:
What does "here" denote? Is the intention here that we are blocking signals so that they will be blocked in the parent process outside the sandbox (the one that calls `monitor_child()`), or so that they will be blocked in the child process inside the sandbox (the one that eventually executes a user-specified *COMMAND*), or what?

If I'm reading correctly, what you are aiming to do is:

* blocking these signals, so that they won't terminate the parent outside the sandbox, and therefore the parent can pass them through to the child inside the sandbox;
* doing the `clone()` to separate parent from child;
* restoring the blocked/unblocked state of these signals in the child, leaving them blocked in the parent

but ideally I wouldn't have to guess this, because it would be clear from the code and/or the commit messages.

`block_forwarded_signals()` would probably be a clearer name for this function.